src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -369,30 +413,45 @@ public ConceptReferencesResult findConceptReferences(
 		return new ConceptReferencesResult(typeSets, conceptReferencesPage.getPageable(), conceptReferencesPage.getTotalElements());
 	}
 
-	@RequestMapping(value = "/browser/{branch}/concepts", method = RequestMethod.POST)
+	@PostMapping(value = "/browser/{branch}/concepts/{validate}")
```

Comment:
I don't mind you changing `@RequestMapping` to `@PostMapping` but it's going to be easier to scan read if this is done consistently. Please either change them for all controllers in snowstorm using a separate refactoring commit or just use the current standard.

Reply:
Fixed by refactoring the whole Controller to use the new annotation introduced in version 4.3 of Spring.

Reply:
I suggested all snowstorm controllers but this is fine.